packages/frontend/src/components/page-not-found/PageNotFound.js

```diff
@@ -44,7 +45,12 @@ const StyledContainer = styled(Container)`
   }
 `;
 
-export function PageNotFound() {
+export function PageNotFound({ history }) {
+    useEffect(() => {
+        recordWalletMigrationEvent('REDIRECT');
```

Comment:
Is it worth reporting the requested path here too? If we care enough about 404s from deprecated paths then that also seems relevant.

Reply:
Correct!, but I noticed that rudder stack does it internally so decided to not include there:
<img width="1045" alt="image" src="https://github.com/near/near-wallet/assets/6027014/d4fd056b-8c5e-4803-8ab3-bd63192ab380">